### DISCUSSION TITLE
react-hook-form and react-dropdown-select

### DISCUSSION THREAD

**Discussion by seenitall** [react-hook-form/react-hook-form#3777] [General]
Hi, I started to take a look to the "react-dropdown-select" library (https://github.com/sanusart/react-dropdown-select - API documentation: https://sanusart.github.io/react-dropdown-select/api), which is supposed to be a more flexible extension of react-select. 

Considering that I would expect it to work with react-hook-form. I attach three examples: an standalone react-dropdown-select, an attempt to make it work with Controller, and the counterpart "react-select" component working with Controller.

https://codesandbox.io/s/epic-feather-4szdr?file=/src/App.js

Any ideas on where to look at?

**Comment by daves28** [comment]
Looks like react-dropdown-select doesn't expose the ref via props, so there's no way to register the input? I'm running into the same problem. Might be worth a PR to react-dropdown-select

**Comment by bluebill1049** [comment]
if ref is not exposed, you can always use `useController`, or `Controller`.

**Comment by daves28** [comment]
Doesn't work. I've tried.

Edit: I apologies. You are correct. 
To the OP, you are not passing the react-hook-form onChange function provided from the render prop into the Select component onChange function. When you do it does work.
